loopd/swapclient_server.go

```diff
@@ -83,8 +89,9 @@ func (s *swapClientServer) LoopOut(ctx context.Context,
 		}
 	}
 
-	// Check that the label is valid.
-	if err := labels.Validate(in.Label); err != nil {
+	sweepConfTarget, err := validateLoopOutRequest(s.lnd.ChainParams,
+		in.SweepConfTarget, sweepAddr, in.Label)
```

Comment:
nit: see lnd [contribution guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#additional-style-constraints-on-top-of-gofmt) for wrapping on functions:

```
validateLoopOutRequest(
s.lnd.ChainParams, in.SweepConfTarget, sweepAddr, in.Label,
)
```

We should add some guidelines for loop 🤔